WalletWasabi.Fluent/ViewModels/Wallets/CoinJoinStateViewModel.cs

```diff
@@ -30,7 +30,7 @@ public partial class CoinJoinStateViewModel : ViewModelBase
 	private const string WaitingRoundMessage = "Waiting for a round";
 	private const string PlebStopMessage = "Coinjoining might be uneconomical";
 	private const string PlebStopMessageBelow = "Receive more funds or press play to bypass";
-	private const string WaitingForConfirmedFundsMessage = "Waiting for confirmed funds";
+	private const string NoCoinsToMixMessage = "No funds available to mix.";
```

Comment:
```suggestion
	private const string NoCoinsToMixMessage = "Not enough funds available to mix.";
```

IMO it should be something like `Not enough funds available to mix.` because a user might have unconfirmed or banned coins or some other reason.